src/components/JdsCard/Card.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <div
+    :class="{
+      'jds-card': true,
+      'jds-card--has-action': showAction,
+      'jds-card--has-media': showMedia,
+      'jds-card--has-thumbnail': showThumbnail,
+      'jds-card--single-button': isSingleButton
+    }"
+  >
+    <div class="jds-card__media">
+      <!-- 
+        @slot Can be used to display any kind of media.
+        Will override `image` prop when used.
+      -->
+      <slot name="media">
+        <img
+          :src="imageSrc"
+          :alt="imageAlt"
+          class="jds-card__media__image-default"
+        />
+      </slot>
+    </div>
+    <div class="jds-card__header">
+      <img
+        :src="thumbnailSrc"
+        :alt="thumbnailAlt"
+        class="jds-card__thumbnail"
+      />
+      <div class="jds-card__titles">
+        <!-- 
+          @slot Will override title and subtitle views
+          provided within corresponding props.
+        -->
+        <slot name="titles">
+          <p class="jds-card__title font-sans-2">
+            {{ title }}
+          </p>
+          <p class="jds-card__subtitle font-sans-2">
+            {{ subtitle }}
+          </p>
+        </slot>
+      </div>
+      <div class="jds-card__action">
+        <!-- 
+          @slot Will override action icon view
+          provided within `action` props.
+        -->
+        <slot name="action">
+          <button
+            class="jds-card__action__icon-default"
+            @click="onClickAction"
+          >
+            <jds-icon
+              v-if="actionIcon"
+              :name="actionIcon"
+              size="18px"
+              fill="#008444"
+            />
+          </button>
+        </slot>
+      </div>
+    </div>
+    <div class="jds-card__body">
+      <!-- 
+        @slot Default slot
+       -->
+      <slot></slot>
+    </div>
+    <div class="jds-card__buttons">
+      <!-- 
+        @slot Will override buttons view
+        provided within `button` props.
+       -->
+      <slot name="buttons">
+        <jds-button
+          v-for="(btn, i) in cardButtons"
+          :key="i"
+          class="jds-card__button-default"
+          :variant="getButtonVariant(i)"
+          @click="onClickButton(btn, i)"
+        >
+          <strong>
+            {{ btn.label }}
+          </strong>
+        </jds-button>
+      </slot>
+    </div>
+  </div>
+</template>
+
+<script>
+import JdsIcon from '../JdsIcon'
+import JdsButton from '../JdsButton'
+export default {
+  components: {
+    JdsIcon,
+    JdsButton
+  },
+  props: {
+    /**
+     * Set image on top of card.
+     */
+    image: {
+      type: [String, Object],
+    },
+    /**
+     * Set thumbnail on the left side of titles.
+     */
+    thumbnail: {
+      type: [String, Object],
+    },
+    /**
+     * Set card title
+     */
+    title: {
+      type: String,
+    },
+    /**
+     * Set card subtitle
+     */
+    subtitle: {
+      type: String,
+    },
+    /**
+     * Set card action icon
+     */
```

Comment:
hello @adrianpdm, I think it would be better to add reference to Jds Icon, so user will not confused about what icon name they should use.

Reply:
thank you, fixed on [this commit](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/56/commits/ecbba0ac770707f093cf61dd1123b1431269baaa)